tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -723,6 +734,36 @@ def validate_header_rpt_month_year(datafile, header, generate_error):
         )
     return is_valid, error
 
+
+def t3_m3_child_validator(which_child):
+    """T3 child validator."""
+    def t3_first_child_validator_func(value, temp, friendly_name, item_num):
+        if not _is_empty(value, 1, 60) and len(value) >= 60:
+            return (True, None)
+        else:
+            return (False, "1st child record truncated.")
```

Comment:
@ADPennington , @reitermb : could you provide the error language you want here?

Reply:
Maybe @victoriaatraft can help here? 

Context: T3 records can have data for one or two children. The system will not process the record if it's truncated/too short. for example, the data for the first child in the record is captured in the first 60 characters, so if the record length is 59 characters, the above message will be produced. Any suggestions for a better message?

Reply:
Perhaps (if it's possible to show how many characters they submitted):
The [first child] record is too short at [XX] characters and must be at least 60 characters.

Reply:
thanks so much @victoriaatraft. 

Is it feasible @raftmsohani? and if so, can we do something similar for child 2 (if the record is >=61 and < 101?

Reply:
Yes, I will apply this change